lib/rpm_package.py

```diff
@@ -103,7 +103,7 @@ def update_version(self, new_version):
 
         self.write_content()
 
-    def bump_release(self, change_log_lines, user_name, user_email):
+    def update_release_and_change_log(self, change_log_lines, user_name, user_email):
```

Comment:
FR> Regardless, I don't see how bumping the release would not cause the change log to be updated or how updating the change log would not cause the release to be bumped.

If the rpm package does not keep a change log, you could simply increment the release number. Also, it is not obvious to a beginner that a rpm spec even has a change log. Several other file descriptors (eg XML/JSON/YAML files, such as our package metadata yaml file) do not. I always recommend not making assumptions and keeping things explicit.